src/Snackbar/Snackbar.react.js

```diff
@@ -68,16 +67,18 @@ function getStyles(props, context) {
             local.message,
             props.style.message,
         ],
-        actionContainer: [
-            snackbar.actionContainer,
-            local.actionContainer,
-            props.style.actionContainer,
-        ],
-        actionText: [
-            snackbar.actionText,
-            local.actionText,
-            props.style.actionText,
-        ],
+        button: {
+            container: [
+                snackbar.button.container,
+                local.button.container,
+                props.style.button.container,
+            ],
+            text: [
+                snackbar.button.text,
+                local.button.text,
+                props.style.button.text,
+            ],
+        },
```

Comment:
Yeah, I was thinking about that, but we also need to remove the horizontal padding and set the height to 14dp on the action. Because as described in the guidelines, there is only a 24dp margin on the left and right side of the action text.

We can't add those styles locally because we want to give other developers the option change it globally in their theme. We would eighter need a to add styles for the action and merge it like so:
```js
button: [
  context.uiTheme.button,
  { container: context.uiTheme.snackbar.action },
  local.button,
  props.button.style,
]
```
Or we need to find and nicer way to merge those action button styles with the default button styles.